_posts/2020-04-28-history-of-bel.md

```diff
@@ -35,30 +35,72 @@ community. And so, we being on a dark and stormy night in 2001....
+Keith was brought in by [Jim Serum](https://www.linkedin.com/in/james-serum-b10ba/), who co-founded [Viaken Systems](https://www.crunchbase.com/organization/viaken-systems) with Elliston, and was on the board of Genstruct.
+
+**October 2002** Keith submits a funding proposal and business plan for Genstruct, based upon a new approach to using artificial intelligence for biological networks proposed by Navin Chandra and developed by the team.
+The board approves the plan, approves investment in Genstruct, and hires Keith as the CEO.
 
 ### 2003
 
-**September 8th, 2003** Selventa [raises $6.5M](https://www.crunchbase.com/funding_round/selventa-series-a--1f3d3583)
+**January 2003** Genstruct does its first commercial project, a pilot project with Pfizer, where they successfully identified the mechanism of action of an unknown cancer drug.
+This leads to a more than 10 year collaboration between the companies.
+
+**Spring 2003**. Biological Expression Language is created by Dexter Pratt, Navin Chandra, Keith Elliston and Ted Slater.
+This development is based on work done by Navin at [Perot Systems and Nets](https://en.wikipedia.org/wiki/Perot_Systems), work done by Dexter on CycL with [Cycorp](https://www.cyc.com), and on biological ontologies by Keith and Ted.
+
+**September 8th, 2003** Genstruct [raises $6.5M](https://www.crunchbase.com/funding_round/selventa-series-a--1f3d3583)
 in a Series A with lead investors [Flagship Pioneering](https://www.flagshippioneering.com/companies/selventa) and
-[A.M. Pappas & Associates](https://www.pappas-capital.com/portfolio/selventa/). Later they will raise $500K and $5M
-in successive Venture Rounds. The link on Flagship Pioneering to this event is [dead](https://www.flagshippioneering.com/about/news/genstruct-completes-65-million-series-financing).
+[A.M. Pappas & Associates](https://www.pappas-capital.com/portfolio/selventa/), during which they acquire the Pappas portfolio company [Incellico, Inc.](https://www.crunchbase.com/organization/incellico-inc) (led by [John Wilbanks](https://www.linkedin.com/in/johnwilbanks/) and [Toby Segaran](https://www.linkedin.com/in/toby-segaran-80bbb9/)).
+
+### 2004
 
-**Sometime in 2003** Biological Expression Language is created by Dexter Pratt while working at Selventa ([ref](https://fairsharing.org/bsg-s000572))
+**2004** Toby Segaren, working with Justin Sun and other developers at the company, develop the first practical version of the Genstruct Inference Engine.
+The inference engine implements a reverse causal reasoning using a graph-based approach, and identifies upstream causes (mechanisms) for downstream observations (state changes).
```

Comment:
Is this the same thing that was later published as RCR / Whistle? Or was this never published?